expression/aggregation/aggregation.go

```diff
@@ -97,6 +99,7 @@ type AggEvaluateContext struct {
 	DistinctChecker *distinctChecker
 	Count           int64
 	Value           types.Datum
+	Extra           types.Datum
```

Comment:
It is better to add a comment for it.

Reply:
done. Extra -> Variance and comments added.